Gordon360/Services/MembershipRequestService.cs

```diff
@@ -265,74 +163,26 @@ public async Task<IEnumerable<MembershipRequestViewModel>> GetMembershipRequests
-        }
-
-        // Helper method to help validate a membership request that comes in.
-        // Return true if it is valid. Throws an exception if not. Exception is cauth in an Exception filter.
-        private bool ValidateMembershipRequest(REQUEST membershipRequest)
-        {
-            var personExists = _context.ACCOUNT.Where(x => x.gordon_id.Trim() == membershipRequest.ID_NUM.ToString()).Any();
-            if (!personExists)
-            {
-                throw new ResourceNotFoundException() { ExceptionMessage = "The Person was not found." };
-            }
-            var activityExists = _context.ACT_INFO.Where(x => x.ACT_CDE.Trim() == membershipRequest.ACT_CDE).Any();
-            if (!activityExists)
-            {
-                throw new ResourceNotFoundException() { ExceptionMessage = "The Activity was not found." };
-            }
-            var participationExists = _context.PART_DEF.Where(x => x.PART_CDE.Trim() == membershipRequest.PART_CDE).Any();
-            if (!participationExists)
-            {
-                throw new ResourceNotFoundException() { ExceptionMessage = "The Participation level was not found." };
-            }
-            var sessionExists = _context.CM_SESSION_MSTR.Where(x => x.SESS_CDE.Trim() == membershipRequest.SESS_CDE).Any();
-            if (!sessionExists)
-            {
-                throw new ResourceNotFoundException() { ExceptionMessage = "The Session was not found." };
-            }
-            // Check for a pending request
-            var pendingRequest = _context.REQUEST.Any(x => x.ID_NUM == membershipRequest.ID_NUM &&
-                x.SESS_CDE.Equals(membershipRequest.SESS_CDE) && x.ACT_CDE.Equals(membershipRequest.ACT_CDE) &&
-                x.STATUS.Equals("Pending"));
-            if (pendingRequest)
-            {
-                throw new ResourceCreationException() { ExceptionMessage = "A request for this activity has already been made for you and is awaiting group leader approval." };
-            }
-
-            return true;
-        }
+            original.SESS_CDE = membershipRequest.SessCode;
+            original.COMMENT_TXT = membershipRequest.CommentText;
+            original.DATE_SENT = membershipRequest.DateSent;
+            original.PART_CDE = membershipRequest.PartCode;
 
-        private bool IsPersonAlreadyInActivity(REQUEST membershipRequest)
-        {
-            var personAlreadyInActivity = _context.MEMBERSHIP.Where(x => x.SESS_CDE == membershipRequest.SESS_CDE &&
-                x.ACT_CDE == membershipRequest.ACT_CDE && x.ID_NUM == membershipRequest.ID_NUM).Any();
-            if (personAlreadyInActivity)
-            {
-                throw new ResourceCreationException() { ExceptionMessage = "You are already part of the activity." };
-            }
+            await _context.SaveChangesAsync();
 
-            return true;
+            return _context.RequestView.FirstOrDefault(r => r.RequestID == original.REQUEST_ID); ;
```

Comment:
Could you use this service's `Get` method here?
```suggestion
            return Get(original.REQUEST_ID);
```